src/Stack/Build/Source.hs

```diff
@@ -315,25 +315,13 @@ loadLocalPackage pp = do
             { packageConfigEnableTests = not $ Set.null tests
             , packageConfigEnableBenchmarks = not $ Set.null benches
             }
-        testconfig = config
-            { packageConfigEnableTests = True
-            , packageConfigEnableBenchmarks = False
-            }
-        benchconfig = config
-            { packageConfigEnableTests = False
-            , packageConfigEnableBenchmarks = True
-            }
```

Comment:
For my benefit, could you explain further why these can be safely deleted? I am not across the architectural considerations and would appreciate some help in understanding them.

Reply:
Honestly I don't know why it existed in the first place, as it's never been used as far as I know.

Reply:
The fields have been there for a very long time  (7+ years) and I don't know at which point they stopped being used.
The reason why its important to remove them is that, we should have dependencies tracked at the component level for mixins/backpack support, not at the LocalPackage/Package level.